src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n + 1) {
```

Comment:
According to BEM methodology, modifiers should not change the structure or positioning of the blocks. Instead, they should modify appearance or behavior. In this case, you're using the modifiers to target child elements, which is not the standard BEM practice. You should apply the modifier to the star elements themselves, rather than using descendant selectors. This will make your code more maintainable and scalable.